modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Select/Select.tsx

```diff
@@ -42,22 +42,46 @@ describe('Select', () => {
 		}));
 	});
 
+	const options = [
+		{
+			label: 'Option 1',
+			reference: 'Option1Reference',
+			value: 'value1',
+		},
+		{
+			label: 'Option 2',
+			reference: 'Option2Reference',
+			value: 'value2',
+		},
+	];
+
+	it('renders data-option-reference in option elements', () => {
+		const props = {
+			multiple: false,
+			name: 'selectName',
+			options,
+		};
+
+		const {getAllByRole, getByRole} = render(
+			<SelectWithProvider {...props} />
+		);
+
+		userEvent.click(getByRole('combobox'));
+
+		const optionElements = Array.from(getAllByRole('option'));
```

Comment:
Hey @cgmonte, could you check if the `getAllByRole` already return an array? Try to remove `Array.from` and check if it works.

I was able to find a usage example on the portal this way:

`getAllByRole('elements').forEach((element) => {})`

Reply:
Hey, you are right. I got this error where `HTMLCollection` wouldn't have array methods earlier in the development cycle, but that was probably with the return of another method, not `getAllByRole`, because this one is an array-like object and has array methods. I have [updated](https://github.com/liferay-forms/liferay-portal/pull/2611/commits/11f0909a7de0171f51b83253556fe9ee7df02321#diff-fa2dbea0aa8d858412d4d8c94daff6fcb283a185ba41c86021e9d97a9ff9deacR71) the code.